### ISSUE TITLE
Configurable mouse speed

### ISSUE THREAD

**Issue by s1hofmann** [sakuli/sakuli#197]
**Short overview**
As a Sakuli user I want to be able to configure mouse movement speed.

**Use case**
Faster test execution.

**Detailed description**
Currently it is not possible to configure mouse movement speed in Sakuli.
Since the default movement speed is rather slow, Sakuli should provide ways to configure movement speed via property.

**Comment by s1hofmann** [comment]
Depends on a change in nut.js, currently on hold.

**Comment by svettwer** [comment]
blocked by https://github.com/nut-tree/nut.js/issues/85

**Comment by svettwer** [comment]
IMHO changing the mouse speed and make it configurable is straight forward.

- `movementtype.function.ts` calculating time slices for mouse movement
- `mouse.class.ts` performing the movement by slicing the time into moving and sleeping actions

There are some tests to alter which might slightly increase the effort.

**Comment by svettwer** [comment]
There could be an alternative to inject the mouse speed config via the `mouse.class.ts` config object via Sakuli

**Comment by svettwer** [comment]
not blocked anymore, https://github.com/nut-tree/nut.js/issues/85 has been solved.

**Comment by svettwer** [comment]
Only nut.js update required

**Comment by cliffle** [comment]
For the nut.js update, we need to update typescript in Sakuli first because nut.js uses typescript 3.8.3 from version 1.3.2.

**Comment by svettwer** [comment]
Hint from @s1hofmann: "Have a look at the KeyboardAPI" + "A property is required to pass the mouse speed to nut-js via sakuli"
